content/solutions/pipeline.adoc

```diff
@@ -29,8 +29,7 @@ features like:
 * checking the pipeline definition into source control (`Jenkinsfile`)
 * support for extending the domain specific language with additional,
   organization specific steps, via the
-  "link:https://www.cloudbees.com/blog/jenkins-workflow-using-global-library-implement-re-usable-function-call-secured-http-endpoint[global
-  library]" feature
```

Comment:
Yeah, at first I just wanted to fix the 404 link, but then I thought it would be better to refer directly to the plugin doc rather than a blog post about one example use of it.